src/aznfswatchdog

```diff
@@ -621,5 +541,40 @@ while :; do
             fi
         fi
     done
+}
+
+vecho $LOG_V3 "Starting aznfswatchdog for NFSv3..."
+
+# Detect and log distro, bash and AZNFS-mount version
+log_version_info
+
+# Dump NAT table once on startup in case we have reported conflicts.
+vecho $LOG_V3 "NAT table:\n$(iptables-save -t nat)"
+conntrack -L > /dev/null
+
+# conntrack timewait timeout higher than the TCP timewait timeout value isn't very valuable.
+conntrack_timeo_timew=$(cat /proc/sys/net/netfilter/nf_conntrack_tcp_timeout_time_wait 2>/dev/null)
+if [ $? -eq 0 ] && [ -n "$conntrack_timeo_timew" -a $conntrack_timeo_timew -gt $AZNFS_TIMEWAIT_TIMEOUT ]; then
+        vecho $LOG_V3 "Changing /proc/sys/net/netfilter/nf_conntrack_tcp_timeout_time_wait [$conntrack_timeo_timew -> $AZNFS_TIMEWAIT_TIMEOUT]"
+        echo $AZNFS_TIMEWAIT_TIMEOUT > /proc/sys/net/netfilter/nf_conntrack_tcp_timeout_time_wait
+fi
+
+if ! chattr -f +i $MOUNTMAPv3; then
+    wecho $LOG_V3 "chattr does not work for ${MOUNTMAPv3}!"
+fi
+
+#
+# If we are not skipping IP change detection for regional accounts, then mountmap file will be updated every minute
+# due to the change in IP for regional account FQDN. If we use standard MOUNTMAP_INACTIVITY_SECS,
+# then this will prevent mountmap entry to be deleted. We set this to less than 1 minute in this case.
+#
+if [ "$AZNFS_SKIP_IP_CHANGE_DETECTION_FOR_REGIONAL_ACCOUNTS" == "0" ]; then
+    MOUNTMAP_INACTIVITY_SECS=45
+fi
+
+while :; do
+    sleep $MONITOR_INTERVAL_SECS
+    vecho $LOG_V3 "Processing NFSv3 mounts..."
```

Comment:
nit: This log feels redundant to me. This will keep repeating each 5 seconad and will fill the log file unnecessarily. This can be removed. 
for both v3 and v4